lib/src/components/NxIconDropdown/stateful/__tests__/NxStatefulIconDropdown.test.tsx

```diff
@@ -5,187 +5,233 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import { act } from 'react-dom/test-utils';
-import { mount, ReactWrapper } from 'enzyme';
-import 'jest-enzyme';
-import * as enzymeUtils from '../../../../__testutils__/enzymeUtils';
+import { rtlRender, rtlRenderElement, userEvent, runTimers } from '../../../../__testutils__/rtlUtils';
+import { act, within } from '@testing-library/react';
 
-import NxButton from '../../../NxButton/NxButton';
-import NxIconDropdown from '../../NxIconDropdown';
 import NxStatefulIconDropdown, { Props } from '../NxStatefulIconDropdown';
 
 describe('NxStatefulIconDropdown', () => {
-  let container: HTMLDivElement | null;
+  const childClickSpy = jest.fn();
```

Comment:
Spies should be initialized in `beforeEach` or in the tests themselves.  `minimalProps` should just use a noop `() => {}` for required function props

Reply:
Updated here c8b692165353a70352f64a1f27f767e82b6e3d22.